packages/server/modules/workspaces/services/tracking.ts

```diff
@@ -170,22 +170,29 @@ export const updateAllWorkspacesTackingPropertiesFactory =
       getWorkspaceSeatCount: deps.getWorkspaceSeatCount
     })
 
-    const bultPropertiesAndPushThenToMixpanel = async (workspace: Workspace) => {
+    const buildPropertiesAndPushThenToMixpanel = async (workspace: Workspace) => {
       mixpanel.groups.set(
         WORKSPACE_TRACKING_ID_KEY,
         workspace.id,
         await buildWorkspaceTrackingProperties(workspace)
       )
     }
 
+    const MAX_ITERATIONS = 100_000
```

Comment:
100,000 seems a bit too high, i'd put it closer to 10,000, but maybe 4000 iterations isn't that bad